src/index.spec.ts

```diff
@@ -1,12 +1,182 @@
+import cases from 'jest-in-case';
+import fetch from 'node-fetch';
+import { mocked } from 'ts-jest/utils';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+jest.mock('node-fetch', () => jest.fn());
+
+describe('isInteger', () => {
+  cases(
+    'should return',
+    (opts) => {
+      expect(isInteger(opts.actual)).toBe(opts.expected);
+    },
+    [
+      { name: 'true evaluating a number', actual: 123, expected: true },
+      {
+        name: 'true evaluating a negative number',
+        actual: -123,
+        expected: true,
+      },
+      { name: 'false evaluating a string', actual: '1', expected: false },
+      { name: 'false evaluating a float', actual: 1.25, expected: false },
+    ],
+  );
```

Comment:
It is more understandable if you use the Jest method separate from each test with the function `it()`

```
 it('should return true evaluating a number'.()=>{...});
 it('should return true evaluating a negative number' () => {...});
 it('should return false evaluating a string', () => {...});
 it('shoudl return false evaluating a floar', () => {...});
```

And it is the same processing because in every case make the call to the function to be testing as in the `it`